t/tasks/inventory/generic/dmidecode/memory.t

```diff
@@ -856,6 +881,7 @@ my %tests = (
             CAPACITY         => '2048',
             CAPTION          => 'J8J2',
             MANUFACTURER     => undef,
+            MODEL            => '0x4B0000000000000000000000000000000000',
```

Comment:
Here using hex2char() will produce a string with wrong null chars. To fix that, calling also getSanitizedString() api is required.